src/pages/home/_what-our-clients-say.js

```diff
@@ -79,92 +55,212 @@ const ClientCard = styled(Flex)`
+        quote: (
+            <Localize translate_text="Deriv is the most reliable broker - excellent customer support and fast payments. It’s a great platform for commodities, forex, and synthetics trading." />
+        ),
+    },
+    {
+        id: 'simon',
+        name: 'Simon',
+        date: '10 September 2021',
+        quote: (
+            <Localize translate_text="Low spreads on Volatility synthetics and fast withdrawals- Deriv is a good broker!" />
+        ),
+    },
+    {
+        id: 'allan',
+        name: 'Allan',
+        date: '14 January 2021',
+        quote: (
+            <Localize translate_text="The app [Deriv P2P] is good. It’s easy to deposit and withdraw for small traders." />
+        ),
+    },
+    {
+        id: 'godfrey',
+        name: 'Godfrey',
+        date: '16 June 2021',
+        quote: <Localize translate_text="This app is a blessing!" />,
+    },
+    {
+        id: 'katleho',
+        name: 'Katleho',
+        date: '8 February 2021',
+        quote: (
+            <Localize translate_text="Wow! Deriv GO is so perfect, it's convenient and reliable. I highly recommend using the app." />
+        ),
+    },
+    {
+        id: 'jackline',
+        name: 'Jackline',
+        date: '13 September 2021',
+        quote: (
+            <Localize translate_text="I've been a trader for many years and I've never encountered a good broker like Deriv before – it's the best for customer care and payment options!" />
+        ),
+    },
+    {
+        id: 'freeman',
+        name: 'Freeman',
+        date: '7 March 2021',
+        quote: <Localize translate_text="DP2P is a flawless innovation." />,
+    },
+    {
+        id: 'vikas',
+        name: 'Vikas',
+        date: '24 May 2021',
+        quote: (
+            <Localize translate_text="It's been a really great experience trading forex on Deriv - it's a smooth and seamless operation!" />
+        ),
+    },
+    {
+        id: 'moyz',
+        name: 'Moyz',
+        date: '7 March 2021',
+        quote: (
+            <Localize translate_text="This app is really good! I've been looking for an app like this which is easy to use." />
+        ),
+    },
+    {
+        id: 'ian',
+        name: 'Ian',
+        date: '23 June 2021',
+        quote: <Localize translate_text="Nice app to buy USD. Fast and efficient!" />,
+    },
+    {
+        id: 'ls',
+        name: 'LS',
+        date: '14 May 2021',
+        quote: (
+            <Localize translate_text="Deriv is the best forex broker I have ever come across!" />
+        ),
+    },
+]
 
-        if (decimal_place) {
-            if (item_value >= 5) {
-                place_value = 5
-            } else {
-                place_value = ''
-            }
-        }
-
-        return place_value
-    })
+const ClientSlide = ({ quote, name, date }) => (
+    <Flex direction="column" height="100%" jc="space-between">
+        <ClientTestimonial size={'24px'} weight={400} lh={'36px'} className="flexi-item">
```

Comment:
If you're using a styled component, you don't need a className.

Reply:
```suggestion
        <ClientTestimonial size="24px" weight="400" lh="36px" className="flexi-item">
```

Reply:
We usually use double quotes for these.

Reply:
For the element the has className it is a special case and require class to be able to apply specific style once it gets passed to the slider to achieve the smooth transition as per the mock up